plugins/katello/nightly/advanced/http_proxies.md

```diff
@@ -0,0 +1,132 @@
+
+To apply the global default proxy policy to one or more products:
+  * navigate to: Content > Products
+  * Select one or more products by clicking the row checkbox for each product you want to assign the http policy for
+  * Click the **Select Action** dropdown
+  * Select **Manage Http Proxy**
+  * Accept the default selection **Global Default (****)". Note that the text within the parenthesis is the name of the Http Proxy set as the global default.
+  * Click **Update**
+
+
+### Selecting the No Http Proxy policy
+
+To choose a policy where no http proxy is used for one or more products:
+  * navigate to: Content > Products
+  * Select one or more products by clicking the row checkbox for each product you want to assign the http policy for
+  * Click the **Select Action** dropdown
+  * Select **Manage Http Proxy**
+  * Select the "No HTTP Proxy" selection.
+  * Click **Update**
+
+### Selecting a Specific Http Proxy
+
+To choose a specific http proxy for one or more products:
+  * navigate to: Content > Products
+  * Select one or more products by clicking the row checkbox for each product you want to assign the http policy for
+  * Click the **Select Action** dropdown
+  * Select **Manage Http Proxy**
+  * Select the "Use specific HTTP proxy" policy selection.
+  * A new drop down will appear with a list of all curently defined http proxies. Select the desired proxy.
+  * Click **Update**
+
+## Selecting a Http Proxy policy and Http Proxy for a new Repository
+
+For a general guideline on creating a new repository see [Creating a Repository](/user_guide/content/index.html). There are 2 fields on the new repostory form for selecting the http proxy policy and, if needed, a specific http proxy.
+
+### Assigning the global default Http Proxy
+
+By default, a new repository will be created with the "Global Default" policy. You can see this is the default selection in the **Http Proxy Policy** selection.
+
+### Assigning the No Http Proxy policy
+
+If you don't want any http proxy to be used, click the **Http Proxy Policy** dropdown and select "No HTTP Proxy".
+
+### Assigning a specific Http Proxy
+
+If you want to assign a specific http proxy to be used for the new repository, click the **Http Proxy Policy**. A new drop down will appear, presenting a list of all currently defined http proxies. Select the desired proxy and the new repository will use that proxy.
+
+## Assigning a Http Proxy Policy and Http Proxy for an existing Repository
+
+To change the http proxy policy or the http proxy used by an existing repository:
+  * navigate to: Content -> Products -> (the repository you want to modify)
+  * under **Sync Settings** cick the form edit icon for the **Http Proxy** field. Note that the default display will show the current http proxy policy and the name of the proxy the policy enables
+  * Dropdowns for both the Http Proxy policy and (if you choose the "Use specific HTTP Proxy" policy) the Http Proxy appear
```

Comment:
s/Dropdowns/Menus/ 
s/and (if you choose the "Use specific HTTP Proxy" policy) the Http Proxy appear/and, if you choose the "Use specific HTTP Proxy" policy, the HTTP Proxy are displayed/